pkg/security/podsecuritypolicy/capabilities/mustrunas_test.go

```diff
@@ -216,23 +214,19 @@ func TestGenerateDrops(t *testing.T) {
 
 func TestValidateAdds(t *testing.T) {
 	tests := map[string]struct {
-		defaultAddCaps   []api.Capability
-		requiredDropCaps []api.Capability
-		allowedCaps      []api.Capability
-		containerCaps    *api.Capabilities
-		shouldPass       bool
+		defaultAddCaps []api.Capability
+		allowedCaps    []api.Capability
+		containerCaps  *api.Capabilities
+		expectedErrors []error
 	}{
 		// no container requests
-		"no required, no allowed, no container requests": {
-			shouldPass: true,
-		},
+		"no required, no allowed, no container requests": {},
 		"no required, allowed, no container requests": {
 			allowedCaps: []api.Capability{"foo"},
-			shouldPass:  true,
 		},
 		"required, no allowed, no container requests": {
 			defaultAddCaps: []api.Capability{"foo"},
-			shouldPass:     false,
+			expectedErrors: []error{errors.New("capabilities: Invalid value: \"null\": required capabilities are not set on the securityContext")},
```

Comment:
WDYT about using back quotes? In this case we don't need to escape double quotes.